pkg/status/status.go

```diff
@@ -406,3 +427,15 @@ func ReportMigration(cfg *rest.Config, mgr manager.Manager, namespace string, ve
 	statusConditions := conditionListBuilder(configv1.OperatorDegraded, configv1.ConditionFalse, msg, "Upgrading")
 	return instance.setStatus(statusConditions)
 }
+
+// getFirstKeyValue returns the first key and value returned while iterating
+// over the given map[string] StatusError.
+// If the map is empty, "" and nil are returned.
+func getFirstKeyValue(m map[string]StatusError) (key string, err StatusError) {
```

Comment:
Nit: the name is a bit misleading because there isn't really a "first" key in a dictionary - or at least that can change depending on what you insert into it?